File: MODUL 2/C/PRAK204-2310817220011-ReginaSilvaMaharatini.c
#include <stdio.h>
#include <math.h>

#define phi 3.142857

int main()
{
    double jari, tinggi;

    printf("Input\n");
    scanf("%lf %lf", &jari, &tinggi);

    double volume = phi * pow(jari, 2) * tinggi;
    double luas = 2 * phi * jari * (jari + tinggi);
    double keliling = 2 * phi * jari;

    printf("\nOutput\n");
    printf("Volume = %.2lf\n", volume);
    printf("Luas = %.2lf\n", luas);
    printf("Keliling = %.2lf\n", keliling);

    return 0;
}

File: MODUL 2/Python/PRAK204-2310817220011-ReginaSilvaMaharatini.py
print("Input")

angka = input().split()
phi = 3.142857

if len(angka) == 1:
    jari = float(angka[0])
    tinggi = float(input())
elif len(angka) == 2:
    jari, tinggi = map(float, angka)

volume = jari**2 * phi * tinggi
luas = 2 * ((phi * jari) * (jari + tinggi))
keliling = 2 * phi * jari

print("\nOutput")
print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")